src/main/kotlin/uk/gov/justice/digital/hmpps/courtregister/resource/CourtMaintenanceResource.kt

```diff
@@ -0,0 +1,111 @@
+package uk.gov.justice.digital.hmpps.courtregister.resource
+
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.fasterxml.jackson.annotation.JsonInclude.Include.NON_NULL
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.media.Content
+import io.swagger.v3.oas.annotations.media.Schema
+import io.swagger.v3.oas.annotations.responses.ApiResponse
+import io.swagger.v3.oas.annotations.security.SecurityRequirement
+import org.springframework.http.MediaType
+import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.validation.annotation.Validated
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.PutMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+import uk.gov.justice.digital.hmpps.courtregister.ErrorResponse
+import uk.gov.justice.digital.hmpps.courtregister.service.CourtService
+import javax.validation.constraints.NotBlank
+import javax.validation.constraints.Size
+
+@RestController
+@Validated
+@RequestMapping("/court-maintenance", produces = [MediaType.APPLICATION_JSON_VALUE])
```

Comment:
these could all live on the Court resource - could have a proper RESTful resource with crud operations

Reply:
Want to protect these resource but not others - hence different context